tests/functions/test_pivot_longer.py

```diff
@@ -1293,3 +1294,37 @@ def test_preserve_extension_types():
     )
 
     assert_frame_equal(expected, actual)
+
+
+def test_dropna_sort_by_appearance():
+    """
+    Test output when `dropna=True` and
+    `sort_by_appearance=True`
```

Comment:
I think it would be a good idea to reference the issue / PR number here in the docstring as well. (either #1168 or #1169 is fine).

Reply:
Sweet. Will do